cve_bin_tool/cli.py

```diff
@@ -588,6 +594,10 @@ def main(argv=None):
         )
         return -1
 
+    if args["use_mirror"] and not args["offline"]:
+        if cvedb_orig.fetch_from_mirror(args["use_mirror"]) == -1:
+            return -1
```

Comment:
I know we literally have a `return -1` above this (I'll file a bug) but it turns out `-1` isn't a valid error code on windows so we need this to be something positive and non-zero.  There's a set of error codes we use here:
https://github.com/intel/cve-bin-tool/blob/main/cve_bin_tool/error_handler.py#L196

maybe just make something called MirrorError then return ERROR_CODES["MirrorError"] or somesuch.